src/components/Settings/AppAuthentication.vue

```diff
@@ -156,6 +156,9 @@ export default {
 	mounted() {
 		this.startEnrollment();
 	},
+	beforeDestroy() {
+		this.lightboxVisible = false;
```

Comment:
This is necessary because KvLightbox emits a close event when it's destroyed, which triggers the completeSetup method in this component. By setting lightbotVisible to false before this component is destroyed, we prevent it from causing a double-navigation error.